app/routes/admin.js

```diff
@@ -0,0 +1,69 @@
+const router = require("express").Router();
+const mongoose = require("mongoose");
+const Request = require("../../models/request");
+const User = require("../../models/users");
+const Donor = require("../../models/donor");
+const bodyParser = require("body-parser");
+
+const authCheck = (req, res, next) => {
```

Comment:
U can have this in a folder named middlewares, export it and use everywhere its required.

Reply:
I've used routes folder for all the routing processes